homeassistant/components/sensibo/quality_scale.yaml

```diff
@@ -0,0 +1,77 @@
+rules:
+  # Bronze
+  config-flow:
+    status: todo
+    comment: |
+      Data descriptions.
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
```

Comment:
Why does `test_flow_get_no_devices` end in an error? I either expect the setup to just succeed because we are able to setup (having dynamic devices would be a requiment for that to be user friendly) or that we abort because the user can't do something about this without restarting

Reply:
Because it's more likely due to a malconfigured device rather than no device at all. But I grant we should expand the text a bit in `strings.json` to explain this further.
To just allow it and pass on would just confuse the user why nothing appears in HA, likewise for abort.

Reply:
Sounds good! Let's note it down but also make sure we end the test in a create entry